ops/ansible/roles/bfd-server/templates/bfd-server.sh.j2

```diff
@@ -33,6 +33,55 @@ export NEW_RELIC_ENVIRONMENT='{{ data_server_new_relic_environment }}'
 {% endif %}
 {% endif %}
 
+{% if data_server_startup_testing is defined and data_server_startup_testing is sameas true %}
+# Begin smoke testing startup routine
+STARTUP_TESTING_RETRIES='{{ data_server_startup_testing_retries | default(3) }}'
+STARTUP_TESTING_BOOT_TIMEOUT='{{ data_server_startup_testing_boot_timeout | default(30) }}'
+STARTUP_TESTING_REQ_TIMEOUT='{{ data_server_startup_testing_req_timeout | default(15) }}'
+STARTUP_TESTING_REQ_BACKOFF_TIMEOUT='{{ data_server_startup_testing_req_backup_timeout | default(10) }}'
+
+check_endpoint() {
+  curl --max-time $STARTUP_TESTING_REQ_TIMEOUT --silent --insecure --cert-type pem --cert "{{ data_server_dir }}/bluebutton-backend-test-data-server-client-test-keypair.pem" "$1" | grep "$2"
```

Comment:
Does this need Bash's `set -o pipefail` in order to work as expected?

Reply:
Technically this should be ok without it, since this is such a short pipe and grep should always fail when curl fails.  Wouldn't hurt to add it though and would catch a weird edge case where the curl fails, but its output still matches the pattern we're looking for.